babel/localtime/_helpers.py

```diff
@@ -19,6 +23,16 @@ def _get_tzinfo(tzenv: str):
     else:
```

Comment:
as `zoneinfo` may be None, should we check for that here?

Reply:
`pytz` is a required dep when Python < 3.9; `zoneinfo` should be available >= 3.9...